code/game/objects/items/toys.dm

```diff
@@ -294,10 +294,37 @@
 	icon = 'icons/obj/singularity.dmi'
 	icon_state = "singularity_s1"
 
+/*
+ * Comic books
+ */
+/obj/item/toy/comic
+	name = "comic book"
+	desc = "A magazine presenting a fictional story through a sequence of images. Perfect for those long, boring shifts."
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "comic"
+	item_state = "comic"
+
+/obj/item/toy/comic/inspector
+	name = "inspector 404 manga"
+	desc = "Inspector 404 follows the adventures of the titular I404, a shell inspector from Konyang. This issue, #67, follows the \
+	404, also known as Kyung-Sun's quest to bring down the corrupt Superintendent Hayashi."
+	icon_state = "comicinspector"
+	item_state = "comicinspector"
+
+/obj/item/toy/comic/stormman
+	name = "stormman manga"
+	desc = "Stormman, often stylized as STORMMAN! is one of Konyang's most beloved anime series, following a masked superhero named Stormman \
+	who has the power to harness the weather to defend his homeworld while keeping his secret identity. Since its release, all kinds of merchanidse \
```

Comment:
```suggestion
	who has the power to harness the weather to defend his homeworld while keeping his secret identity. Since its release, all kinds of merchandise \
```